tests/ut_gamemaker.py

```diff
@@ -72,52 +71,67 @@ def test_remove_multiple_landmarks_to_one(self):
         self.maker1.remove_landmark(["land"])
         self.assertEqual(self.maker1.display_landmarks(), "land1\n", "Did not remove landmarks")
 
+
 class TestMakerCheckStatus(TestCase):
     def setUp(self):
         HuntUser.objects.all().delete()
+        Landmark.objects.all().delete()
+        lm = Landmark(name="dummy", clue="dummy", question="dummy", answer="dummy", order_num=-1)
+        lm.save()
         self.game_maker = gamemaker.GameMaker()
 
     def test_single_team(self):
```

Comment:
Going to have to add "test_single_team_check_status" or something different than the current name to maintain testing integrity. This is what was causing my issues.